### ISSUE TITLE
Old-style channel view 

### ISSUE THREAD

**Issue by kroese** [apotdevin/thunderhub#460]
I just upgraded Thunderhub from 12.31 to 13.13. 

And the most obvious change for me is that the channel overview has transformed in a list/grid.

For desktop users this might be a good thing, but for mobile usage I very much prefered the old layout where you just saw cards which you could expand or collapse when needed. The new layout requires me to scroll from left to right all the time, instead of up and down. 

Is there any way to get the previous layout system back? The new one really feels like a downgrade for mobile users.

**Comment by abhiShandy** [comment]
Makes sense. The channels page should have a mobile-friendly design. We can use designs similar to the transactions page.
